### DISCUSSION TITLE
I'm unable to renew my student developer program

### DISCUSSION THREAD

**Discussion by tmvkrpxl0** [community/community#52700] [GitHub Education]
Hello, I've applied for student developer program in 2021. Back then my school was not listed and I put incorrect expiration date, casuing it to expire sooner. Today my school is indeed listed and I've registered my school email as it is now requirement.
However all of my requests have been denied. I've sent both enrollment paper and my school id card several times but to no avail. I've already put my real name in my account quite a while ago, and I don't see any more reasons that why it should fail.
Is it because all of my documents are in Korean? None of those have english name of my school, but I don't see if I can get english version of it

**Comment by tmvkrpxl0** [comment]
Renaming account to english worked

**Reply by aidarbekq** [reply]
> Renaming account to english worked

renaming account?

**Reply by tmvkrpxl0** [reply]
> > Renaming account to english worked
> 
> renaming account?

Ah, sorry, I meant renaming profile name
